From 3a2fb2d0b816307179c00d37263797c902618e88 Mon Sep 17 00:00:00 2001
From: Yasufumi Ogawa <ogawa.yasufumi@lab.ntt.co.jp>
Date: Sun, 13 Jan 2019 16:08:43 +0900
Subject: [PATCH 3/6] spp_vf/common: add RTE_LOGTYPE in each of files

Define RTE_LOGTYPE_APP was removed from `common.h` to diversify
notation of log messages. This update is to define RTE_LOGTYPE_APP
in each of source files.

Signed-off-by: Yasufumi Ogawa <ogawa.yasufumi@lab.ntt.co.jp>
---
 src/vf/common/command_dec.c  | 1 +
 src/vf/common/command_proc.c | 1 +
 2 files changed, 2 insertions(+)

diff --git a/src/vf/common/command_dec.c b/src/vf/common/command_dec.c
index b328ca8..61dd4f4 100644
--- a/src/vf/common/command_dec.c
+++ b/src/vf/common/command_dec.c
@@ -12,6 +12,7 @@
 #include "command_dec.h"
 
 #define RTE_LOGTYPE_SPP_COMMAND_PROC RTE_LOGTYPE_USER1
+#define RTE_LOGTYPE_APP RTE_LOGTYPE_USER2
 
 /* command string  */
 #define SPP_COMMAND_CLASSFIER_TABLE_STR "classifier_table"
diff --git a/src/vf/common/command_proc.c b/src/vf/common/command_proc.c
index 00ecdc1..0e5c5f1 100644
--- a/src/vf/common/command_proc.c
+++ b/src/vf/common/command_proc.c
@@ -22,6 +22,7 @@
 #include "command_proc.h"
 
 #define RTE_LOGTYPE_SPP_COMMAND_PROC RTE_LOGTYPE_USER1
+#define RTE_LOGTYPE_APP RTE_LOGTYPE_USER2
 
 /* request message initial size */
 #define CMD_RES_ERR_MSG_SIZE  128
-- 
2.17.1

